app/actions/ClientActions.js

```diff
@@ -144,9 +148,9 @@ function getNetworkSuccess(getNetworkResponse) {
   var network = getNetworkResponse.getActiveNetwork();
   // XXX remove network magic numbers here
   var networkStr = '';
-  if (network == 118030347) {
+  if (network == 1223139429) {
```

Comment:
Could you define this numbers somewhere rather than just using the magic numbers directly?  Ideally someplace we can reuse.